organization/_posts/2019/2019-01-30-packaging.md

```diff
@@ -0,0 +1,117 @@
+---
+title: "HSF Packaging Group Meeting #31, January 30, 2019"
+layout: meetings
+---
+
+# {{page.title}}
+
+Agenda
+=======
+[*https://indico.cern.ch/event/790021/*](https://indico.cern.ch/event/790021/)
+
+Participants: Graeme Stewart, Chris Burr, Guilherme Amadio, Ben Morgan, Marco Clemencic, Patrick Gartung, Chris Green, Geri Ganis, Pere Mato, Henry S, Ben Couturier.
```

Comment:
Henry Schreiner

Reply:
Thanks! I've just pushed the update, so o.k. to merge once Travis is through?

Reply:
Indeed!